### DISCUSSION TITLE
Ability to send test email from Utilities as well as in Settings > Email

### DISCUSSION THREAD

**Discussion by paulgoodfield** [craftcms/cms#15194] [Ideas] [answered]
Currently, the only way to send a test email is from Settings > Email. Since most sites using project config have Settings disabled on non-dev environments there is no way to send a test email from these, even though they'll likely have different SMTP settings than the dev environment.

If there was also an option in Utilities to send a test email then you'd be able to check the mail settings are still working on these non-dev environments.

**Comment by tommysvr** [comment, marked as answer]
Craft 5.6.0 made it possible to send test emails even when `allowAdminChanges` is disabled, as Settings will show in read only mode: https://craftcms.com/docs/5.x/system/control-panel.html#read-only-mode
